components/dashboard/src/prebuilds/list/PrebuildTable.tsx

```diff
@@ -0,0 +1,133 @@
+                        }}
+                    />
+                    <Select
+                        value={filter?.status ?? "any"}
+                        onValueChange={(status) => {
+                            if (status === "any") {
+                                onFilterChange({ ...filter, status: undefined });
+                                return;
+                            }
+                            onFilterChange({ ...filter, status: status as StatusOption });
+                        }}
+                    >
+                        <SelectTrigger className="w-[120px]">
+                            <SelectValue placeholder="Status" />
+                        </SelectTrigger>
+                        <SelectContent>
+                            <SelectItem value="any">All</SelectItem>
+                            <SelectItem value="failed">Failing</SelectItem>
+                            <SelectItem value="succeeded">Successful</SelectItem>
+                            <SelectItem value="unfinished">In progress</SelectItem>
+                        </SelectContent>
+                    </Select>
+                </div>
+            </div>
+            <div className="relative w-full overflow-auto mt-4">
+                {prebuilds.length > 0 ? (
+                    <Table>
+                        <TableHeader>
+                            <TableRow>
+                                <TableHead className="w-52">Repository</TableHead>
+                                <TableHead hideOnSmallScreen>Commit</TableHead>
+                                <SortableTableHead
+                                    columnName={sort.sortBy}
+                                    sortOrder={sort.sortOrder}
+                                    onSort={onSort as SortCallback}
+                                    className="w-40"
+                                    hideOnSmallScreen
+                                >
+                                    Triggered
+                                </SortableTableHead>
+                                <TableHead className="w-24">Status</TableHead>
+                                {/* Action column, loading status in header */}
+                                <TableHead className="w-24 text-right">
+                                    {isSearching && (
+                                        <div className="flex flex-right justify-end items-center">
+                                            <LoadingState delay={false} size={16} />
+                                        </div>
+                                    )}
+                                </TableHead>
+                            </TableRow>
+                        </TableHeader>
+                        <TableBody>
+                            {prebuilds.map((configuration) => (
+                                <RepositoryListItem key={configuration.id} prebuild={configuration} />
```

Comment:
nit (non-blocking): based on the selection, the row jump a bit left/right due to different length of the "Status" column. A min-width would fix it I guess.
Can be in a followup. :+1:

Reply:
@geropl does anything jump for you? I don't see any difference in longer and shorter column names 🤔. We already set an explicit width of `6 rem`, which should take care of it.
![image](https://github.com/gitpod-io/gitpod/assets/29888641/8bb9f558-aa87-46d6-a7e9-f3e95c970159)